src/site/content/en/blog/generic-sensor/index.md

```diff
@@ -0,0 +1,523 @@
+it is measuring the force of the table pushing the device upwards. If you push the device to the
+right, acceleration on the X axis would be positive, or negative if the device is accelerated from
+the right toward the left.
+
+Accelerometers can be used for things like: step counting, motion sensing, or simple device
+orientation. Quite often, accelerometer measurements are combined with data from other sources to
+create fusion sensors, such as, orientation sensors.
+
+The
+[`LinearAccelerationSensor`](https://developer.mozilla.org/en-US/docs/Web/API/LinearAccelerationSensor)
+measures acceleration that is applied to the device hosting the sensor, excluding the contribution
+of gravity. When a device is at rest, for instance lying flat on the table, the sensor would measure
+≈ 0 m/s<sup>2</sup> acceleration on three axes.
+
+### Gravity sensor {: #gravity-sensor }
+
+It is already possible for users to manually derive readings close to those of a gravity sensor by
+manually inspecting `Accelerometer` and `LinearAccelerometer` readings, but this can be cumbersome
+and depend on the accuracy of the values provided by those sensors. Platforms such as Android can
+provide gravity readings as part of the operating system, which should be cheaper in terms of
+computation, provide more accurate values depending on the user's hardware, and be easier to use in
+terms of API ergonomics. The
+[`GravitySensor`](https://w3c.github.io/accelerometer/#gravitysensor-interface) returns the effect
+of acceleration along the device's X, Y, and Z axis due to gravity.
+
+{% Aside %} As of February 2021, GravitySensor is still
+[under development](https://bugs.chromium.org/p/chromium/issues/detail?id=1163993) with no estimated
+release date. {% endAside %}
+
+### Gyroscope {: #gyroscope-sensor }
+
+ <figure class="w-figure">
+  {% Video src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/7VItzZMC9Rb2QglsE3s5.mp4", width="800" %}
+  <figcaption class="w-figcaption">
+    Gyroscope sensor measurements
+  </figcaption>
+</figure>
+
+The [`Gyroscope`](https://developer.mozilla.org/en-US/docs/Web/API/Gyroscope) sensor measures
+angular velocity in radians per second around the device's local X, Y, and Z axis. Most consumer
+devices have mechanical ([MEMS](https://en.wikipedia.org/wiki/Microelectromechanical_systems))
+gyroscopes, which are inertial sensors that measure rotation rate based on
+[inertial Coriolis force](https://en.wikipedia.org/wiki/Coriolis_force). MEMS gyroscopes are prone
+to drift that is caused by the sensor's gravitational sensitivity which deforms the sensor's
+internal mechanical system. Gyroscopes oscillate at relative high frequencies, e.g., 10s of kHz, and
+therefore, might consume more power compared to other sensors.
+
+### Orientation sensors {: #orientation-sensors }
+
+<figure class="w-figure">
+  {% Video src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/rhpW784mCvR78nwg6rd1.mp4", width="800" %}
```

Comment:
```suggestion
  {% Video src="video/8WbTDNrhLsU0El80frMBGE4eMCD3/rhpW784mCvR78nwg6rd1.mp4", width="800", autoplay="true" %}
```

Reply:
I should have said `loop` as well sorry ;)